database/step.go

```diff
@@ -126,6 +126,32 @@ func (c *client) GetStepImageCount() (map[string]float64, error) {
 	return counts, err
 }
 
+// GetStepStatusCount gets a count of an images occurance in the database.
+func (c *client) GetStepStatusCount() (map[string]float64, error) {
+	logrus.Tracef("Counting the total of each status for steps in the database")
```

Comment:
I believe this could be just `Trace` since we aren't passing any formatting to the string